packages/sui-test/src/descriptor-environment-patcher.js

```diff
@@ -0,0 +1,85 @@
+const colors = require('colors')
+// List of test describers to be patched.
+const functionsToPatch = ['describe', 'describe.only', 'it', 'it.only']
+// List of environments
+const environments = {
+  CLIENT: 'Client',
+  SERVER: 'Server',
+}
+const isNode = typeof process === 'object' && process.toString() === '[object process]'
+
+export const descriptorsByEnvironmentPatcher = function descriptorsByEnvironmentPatcher () {
+  /**
+   * This function is the one with the purpose of handle and return the function that should be attached to our only{client | server} patch.
+   * @param {Object} descriptor Is the object that contains the name of the descriptor base function and in the cases that we have nested function calls the nested name too
+   * @param {String} env Can be one of the constants defined in the environments object - {SERVER, CLIENT}
+   * @param {Boolean} isOnly A boolean function to know if we are trying to run a descriptor with the .only Function
+   */
+  function buildFunctionForEnv ({descriptorName, firstLevelFnName} = {}, env, isOnly) {
```

Comment:
Don´t merge naming and positional parameters styles.

Use naming parameters over positionals